pkg/template/template_test.go

```diff
@@ -0,0 +1,53 @@
+package template_test
+
+import (
+	"embed"
+	"testing"
+
+	"github.com/codeready-toolchain/toolchain-common/pkg/template"
+	"github.com/codeready-toolchain/toolchain-common/pkg/test"
+	"github.com/stretchr/testify/require"
+)
+
+//go:embed testTemplates/*
+var EFS embed.FS
+
+//go:embed testTemplates/host/*
+var hostFS embed.FS
+
+func TestLoadObjectsFromEmbedFS(t *testing.T) {
+	t.Run("loads objects recursively from all subdirectories", func(t *testing.T) {
+		// when
+		objects, err := template.LoadObjectsFromEmbedFS(&EFS, &template.TemplateVariables{Namespace: test.HostOperatorNs})
+		// then
+		require.NoError(t, err)
+		require.NotNil(t, objects)
+		// we expect to have 4 objects loaded from the test templates folders
+		require.Equal(t, 4, len(objects), "invalid number of expected objects")
```

Comment:
could you verify that the number of objects is different for the host & member folders?